packages/block-library/src/image/image.js

```diff
@@ -188,7 +189,8 @@ export default function Image( {
 		}
 
 		window
-			.fetch( url )
+			// Avoid cache, which seems to help avoid CORS problems.
+			.fetch( url.includes( '?' ) ? url : url + '?' )
```

Comment:
How reliable is this going to be?

Reply:
More reliable than previously. Any images it fails on?

Reply:
Can you provide a bit more info how that helps with CORS? Is there something we can do with [mode: 'cors', // no-cors, *cors, same-origin](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)?

Reply:
It works by bypassing the cache as commented :)